coverart_redirect/server.py

```diff
@@ -59,9 +60,14 @@ class Server(object):
     def __init__(self, config):
         self.config = config
         self.engine = sqlalchemy.create_engine(self.config.database.create_url())
+        self.rate_limiter = RateLimiter(config)
 
     @Request.application
     def __call__(self, request):
+        rate_limiter_response = self.rate_limiter.check_rate_limit('frontend ip=%s' % request.remote_addr);
```

Comment:
I'm guessing this should be changed to something new including the UA, e.g. `caa ua=`?